src/extension.ts

```diff
@@ -35,7 +37,9 @@ function searchAll() {
     var query = getSelectedText(editor);
 
     var dash = new Dash();
-    exec(dash.getCommand(query));
+    var OS = platform();

```

Comment:
Yea definitely this. Globals are undesired, but I think in this case justifiable.